addons/web/static/src/env.js

```diff
@@ -79,7 +79,7 @@ export async function startServices(env) {
             // Keep it simple!
             return;
         }
-        if (toStart.size) {
+        if (toStart.size && !Array.from(toStart).find((service) => service.name === name)) {
```

Comment:
If we have a lot of services, this can be very costly to parse the array as many times, as this is `O(n²)`:
At worst could have 1 "UPDATE" per service, and each of the `n` service loops up to `n` elements, so `O(n²)`.

I would turn `toStart` from `Set` to `Map`, so that checking presence of named service is `O(1)`, so at worst it's `O(n)`.

On runbot with all modules installed, there's only 68 services. So this is only a 68 vs 4624... But that's still 67 times less efficient that with Map :D